flow/flow.cpp

```diff
@@ -23,8 +23,26 @@
 
 #include <opm/simulators/flow/Main.hpp>
 
+#include <stdio.h>
+#include <execinfo.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+void handler(int sig) {
+  void *array[10];
+  size_t size;
+  // get void*'s for all entries on the stack
+  size = backtrace(array, 10);
+  // print out all the frames to stderr
```

Comment:
Oh, sorry, this is not intentional. This is some code to help to parallel debugging. I am removing them now.